docs/devguide/repl.rst

```diff
@@ -4,42 +4,90 @@
 Accessing the REPL
 ==================
 
-Accessing the REPL on the micro:bit requires:
+REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
+in real-time as you write it.
 
-    * Using a serial communication program
-    * Determining the communication port identifier for the micro:bit
-    * Establishing communication with the correct settings for your computer
+Accessing the REPL on the micro:bit will require you to:
 
-If you are a Windows user you'll need to install the correct drivers. The
+    * Determine the communication port identifier for the micro:bit
+    * Use a program to establish communication with the device
+
+If you are a Windows user you'll need to install the correct drivers, the
```

Comment:
Since this was originally written the driver situation is abit better. Most users on Windows 10 won't need to do this, and quite possibly Windows 8.1 users might be also okay (although we get mixed reports) so we could say instead:

```
For versions of Windows before 10 you might need to install the Mbed serial driver, the...
```